cogs/events.py

```diff
@@ -1,21 +1,54 @@
 import traceback
 
-from discord import Color, Embed, Message
+from discord import Color, Embed, Message, Guild, Member
 from discord.ext.commands import Bot, Cog
 
+import re
+import aiohttp
+
 
 class Custom(Cog):
     def __init__(self, bot: Bot) -> None:
         self.bot = bot
 
+    @staticmethod
+    def get_link_code(string: str) -> str:
+        return string.split("/")[-1]
+
+    @staticmethod
+    async def check_our_invite(full_link: str, guild: Guild) -> bool:
```

Comment:
`check_our_invite` can be a bit missleading `is_our_invite` is a better name.
Note that you'll need to rename all references to this function too (line: 50)
```suggestion
    async def is_our_invite(full_link: str, guild: Guild) -> bool:
```